.gitignore

```diff
@@ -3,7 +3,7 @@ WcaOnRails/app/views/regulations/*.*
 WcaOnRails/app/views/regulations/translations/*
 WcaOnRails/app/views/regulations/scrambles/*
 WcaOnRails/app/views/regulations/history/*
-regulations/
+/regulations/
```

Comment:
Just curious, is there a reason you had to change this? I thought they would behave the same.

Reply:
Actually leading `/` indicates project root directory. Without that, it matches directory of the given name anywhere in the tree. In this case we want to ignore the top level regulations directory, but end up ignoring `WcaOnRails/app/views/regulations` as well.
We do have gitignore exceptions below, but if you ignore an entire directory then git don't really care about those exceptions, so we would need to `git add -f` every new file in that directory.
[Reference](https://git-scm.com/docs/gitignore/2.3.0):
> An optional prefix "!" which negates the pattern; any matching file excluded by a previous pattern will become included again. It is not possible to re-include a file if a parent directory of that file is excluded.